Performance control of embedded content/explainer.md

```diff
@@ -0,0 +1,157 @@
+
+#### Open question: how to evolve best practices?
+Performance categories introduced in this proposal are based on the idea of taking the burden of determining performance best practices off of individual site and app developers. However, best practices evolve with time, and for these policies to achieve their goal, the criteria needs to evolve with them. Changing the criteria for what constitutes a policy violation would introduce compatibility issues for anyone opting-in, as things which are allowed today might not be allowed in the future. We need to define a mechanism in which this evolution can happen in a controlled manner, or decide whether it's a reasonable trade off for developers opting in to be expected to keep up with the platform as best practices evolve.
+
+#### Open question: reporting 3pp violations to embedder
+Embedders are best equipped to influence change in the performance when they are aware of where the issues are. While Document Policy provides a Reporting API integration, this only reports violations to the endpoint of the document where the violation occurs. Embedders do not receive reports that the embedded content has incurred policy violations, which is a limitation. Currently under consideration: send a minimal report to the embedder when a violation occurs in the embedded document.
+
+#### Open question: interaction with Heavy Ad Interventions
+
+#### Open question: required policy and report-only mode
+It is unclear from the Document Policy explainer whether a report-only header in an embedded document satifies the requirements set by `Sec-Required-Document-Policy` header.
+
+### Security and Privacy Considerations
+
+#### Global budgets and side-channel attacks
+The criteria proposed in this document includes budgets which are shared globally across documents. This could allow for documents to learn things about cross-origin documents, as described in the [Never Slow Mode explainer](https://github.com/slightlyoff/never_slow_mode?tab=readme-ov-file#global-limits). We consider the same alternatives as NSM as viable for this proposal:
+  * Require CORS
+  * Fuzzy budgeting
+
+#### Frame depth
+As called out in the [Never Slow Mode explainer](https://github.com/slightlyoff/never_slow_mode?tab=readme-ov-file#global-limits), a limit greater than 2 would expose the depth in the treee of a document.
+
+### Dependencies on non-stable features
+* Document Policy ([explainer](https://wicg.github.io/document-policy/))
+  - general: currently, only implemented by Chromium-based browsers.
+  - Document Policy negotiation: disabled by default.
+  - `policy` attribute: currently unimplemented.
+
+### Alternatives considered
+
+#### Custom attributes and headers
+Using Document Policy for this proposal has limitations and challenges, including 3pp violation reporting, opt-in requirement, budget-based state leaks. A custom mechanism was briefly considered, but the following was determined for such approach:
```

Comment:
For those unfamiliar (like myself) it may be worth going into more detail on what a custom mechanism is and what that would look like in comparison

Reply:
Added details.